arch/risc-v/src/mpfs/mpfs_i2c.c

```diff
@@ -972,6 +950,17 @@ struct i2c_master_s *mpfs_i2cbus_initialize(int port)
       return (struct i2c_master_s *)priv;
     }
 
+#ifdef CONFIG_MPFS_COREI2C
```

Comment:
To tidy this up the code could be made into static function but this is fine as well

Reply:
You are right.. this went to upstream already, so merging it like this...